docs/zello-sdk-documentation/index-all.html

```diff
@@ -491,21 +491,30 @@ <h2 class="title">O</h2>
 <div class="block">
      Called when the state of either the <a href="com/zello/sdk/MessageOut.html" title="class in com.zello.sdk"><code>MessageOut</code></a> or <a href="com/zello/sdk/MessageIn.html" title="class in com.zello.sdk"><code>MessageIn</code></a> changes.</div>
 </dd>
+<dt><span class="memberNameLink"><a href="com/zello/sdk/Events.html#onMicrophonePermissionNotGranted--">onMicrophonePermissionNotGranted()</a></span> - Method in interface com.zello.sdk.<a href="com/zello/sdk/Events.html" title="interface in com.zello.sdk">Events</a></dt>
+<dd>
+<div class="block">
+     Called when an invocation of the <a href="com/zello/sdk/Zello.html#beginMessage--"><code>beginMessage()</code></a> method failed because the microphone permission hasn't been granted.</div>
+</dd>
 <dt><span class="memberNameLink"><a href="com/zello/sdk/Events.html#onSelectedContactChanged--">onSelectedContactChanged()</a></span> - Method in interface com.zello.sdk.<a href="com/zello/sdk/Events.html" title="interface in com.zello.sdk">Events</a></dt>
 <dd>
 <div class="block">
      Called when the selected contact changes.</div>
 </dd>
 <dt><span class="memberNameLink"><a href="com/zello/sdk/Zello.html#openMainScreen--">openMainScreen()</a></span> - Method in class com.zello.sdk.<a href="com/zello/sdk/Zello.html" title="class in com.zello.sdk">Zello</a></dt>
 <dd>
-<div class="block">Opens the main screen of the standard ZelloWork app.</div>
+<div class="block">Opens the main screen of the ZelloWork app.</div>
 </dd>
 </dl>
 <a name="I:R">
 <!--   -->
 </a>
 <h2 class="title">R</h2>
 <dl>
+<dt><span class="memberNameLink"><a href="com/zello/sdk/Zello.html#requestVitalPermissions--">requestVitalPermissions()</a></span> - Method in class com.zello.sdk.<a href="com/zello/sdk/Zello.html" title="class in com.zello.sdk">Zello</a></dt>
```

Comment:
We recognize that dangerous is Android's terminology for these permissions, but for Zello these permissions are much more vital to having the app function properly than they are dangerous :-)